### PR TITLE
fix(storage): use lower case in error messages

### PR DESCRIPTION
**Pull request by clems4ever**

### CODE REVIEW

**Review by james-d-elliott — APPROVED** [review]
LGTM

**Review by james-d-elliott — CHANGES_REQUESTED** [review]
More of a question actually instead of a request for changes, but wanted to make sure it's seen. I think there is a difference in returned errors and logged messages, do we want to affect both?

**Comment by clems4ever** [comment]
> More of a question actually instead of a request for changes, but wanted to make sure it's seen. I think there is a difference in returned errors and logged messages, do we want to affect both?

I was mainly fixing the warnings in the IDE due to upper case in error messages but yes, why not also do the logs. I'll update the PR.

**Comment by james-d-elliott** [comment]
> 
> 
> > More of a question actually instead of a request for changes, but wanted to make sure it's seen. I think there is a difference in returned errors and logged messages, do we want to affect both?
> 
> I was mainly fixing the warnings in the IDE due to upper case in error messages but yes, why not also do the logs. I'll update the PR.

Ah makes sense.

**Review by james-d-elliott — APPROVED** [review]

File: internal/storage/mysql_provider.go [hunk 1]
```diff
@@ -70,11 +70,11 @@ func NewMySQLProvider(configuration schema.MySQLStorageConfiguration) *MySQLProv
 
 	db, err := sql.Open("mysql", connectionString)
 	if err != nil {
-		provider.log.Fatalf("Unable to connect to SQL database: %v", err)
+		provider.log.Fatalf("unable to connect to SQL database: %v", err)
```
Comment (nightah) [inline internal/storage/mysql_provider.go:73]:
```suggestion
		provider.log.Fatalf("Unable to connect to SQL database: %v", err)
```

File: internal/storage/mysql_provider.go [hunk 2]
```diff
@@ -70,11 +70,11 @@ func NewMySQLProvider(configuration schema.MySQLStorageConfiguration) *MySQLProv
...
 	}
 
 	if err := provider.initialize(db); err != nil {
-		provider.log.Fatalf("Unable to initialize SQL database: %v", err)
+		provider.log.Fatalf("unable to initialize SQL database: %v", err)
```
Comment (nightah) [inline internal/storage/mysql_provider.go:77]:
```suggestion
		provider.log.Fatalf("Unable to initialize SQL database: %v", err)
```

File: internal/storage/postgres_provider.go [hunk 3]
```diff
@@ -77,11 +77,11 @@ func NewPostgreSQLProvider(configuration schema.PostgreSQLStorageConfiguration)
 
 	db, err := sql.Open("pgx", connectionString)
 	if err != nil {
-		provider.log.Fatalf("Unable to connect to SQL database: %v", err)
+		provider.log.Fatalf("unable to connect to SQL database: %v", err)
```
Comment (nightah) [inline internal/storage/postgres_provider.go:80]:
```suggestion
		provider.log.Fatalf("Unable to connect to SQL database: %v", err)
```

File: internal/storage/postgres_provider.go [hunk 4]
```diff
@@ -77,11 +77,11 @@ func NewPostgreSQLProvider(configuration schema.PostgreSQLStorageConfiguration)
...
 	}
 
 	if err := provider.initialize(db); err != nil {
-		provider.log.Fatalf("Unable to initialize SQL database: %v", err)
+		provider.log.Fatalf("unable to initialize SQL database: %v", err)
```
Comment (nightah) [inline internal/storage/postgres_provider.go:84]:
```suggestion
		provider.log.Fatalf("Unable to initialize SQL database: %v", err)
```

File: internal/storage/sqlite_provider.go [hunk 5]
```diff
@@ -47,11 +47,11 @@ func NewSQLiteProvider(path string) *SQLiteProvider {
 
 	db, err := sql.Open("sqlite", path)
 	if err != nil {
-		provider.log.Fatalf("Unable to create SQL database %s: %s", path, err)
+		provider.log.Fatalf("unable to create SQL database %s: %s", path, err)
```
Comment (nightah) [inline internal/storage/sqlite_provider.go:50]:
```suggestion
		provider.log.Fatalf("Unable to create SQL database %s: %s", path, err)
```

File: internal/storage/sqlite_provider.go [hunk 6]
```diff
@@ -47,11 +47,11 @@ func NewSQLiteProvider(path string) *SQLiteProvider {
...
 	}
 
 	if err := provider.initialize(db); err != nil {
-		provider.log.Fatalf("Unable to initialize SQL database %s: %s", path, err)
+		provider.log.Fatalf("unable to initialize SQL database %s: %s", path, err)
```
Comment (nightah) [inline internal/storage/sqlite_provider.go:54]:
```suggestion
		provider.log.Fatalf("Unable to initialize SQL database %s: %s", path, err)
```

File: internal/storage/sqlmock_provider.go [hunk 7]
```diff
@@ -47,7 +47,7 @@ func NewSQLMockProvider() (*SQLMockProvider, sqlmock.Sqlmock) {
 	db, mock, err := sqlmock.New()
 
 	if err != nil {
-		provider.log.Fatalf("Unable to create SQL database: %s", err)
+		provider.log.Fatalf("unable to create SQL database: %s", err)
```
Comment (nightah) [inline internal/storage/sqlmock_provider.go:50]:
```suggestion
		provider.log.Fatalf("Unable to create SQL database: %s", err)
```

**Review by nightah — CHANGES_REQUESTED** [review]
For consistency let's not lower case our log messages.

**Review by james-d-elliott — APPROVED** [review]
LGTM

**Pull request merged**